### DISCUSSION TITLE
Clipboard triggers

### DISCUSSION THREAD

**Discussion by honeymath** [espanso/espanso#1756] [Feature requests and Ideas]
Monitor the clipboard, if some thing matches to the predefined regular expression, then do something.  This feather is inspired by PhraseExpress. This feature is extremely useful when interacting with online web chatgpt, as it always generate response to copy, I hope whenever I copied, it automatically load some application. Second consideration is expansion by copying keyword.

**Comment by AucaCoyan** [comment]
That's is clever! I love this feature request
